helm/dagster/values.schema.json

```diff
@@ -12,6 +12,10 @@
         "postgresql": {
             "$ref": "#/definitions/PostgreSQL"
         },
+        "generatePostgresqlPasswordSecret": {
+            "title": "Generatepostgresqlpasswordsecret",
+            "type": "boolean"
+        },
```

Comment:
this json file is a generated artifact, 
the source of truth for this is https://github.com/dagster-io/dagster/blob/master/helm/dagster/schema/schema/charts/dagster/values.py

the cli for generating is at `helm/dagster/schema/schema/cli.py`

Reply:
Oh, cool -- fixed, thanks!